src/main/java/io/confluent/connect/elasticsearch/ElasticsearchWriter.java

```diff
@@ -53,6 +54,7 @@
 
   private final Set<String> existingMappings;
   private final BehaviorOnMalformedDoc behaviorOnMalformedDoc;
+  private final ErrantRecordReporter reporter;
```

Comment:
this member variable (like the one above) is unused in this class. It's used in the `BulkProcessor` instead.